services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -1907,17 +1906,14 @@ class HakemusServiceITest(
         }
 
         @Test
-        fun `sends email to the orderer when hakemus gets a decision`() {
+        fun `sends email to the contacts when hakemus gets a decision`() {
             val hanke = hankeFactory.saveMinimal()
-            applicationRepository.save(
-                ApplicationFactory.createApplicationEntity(
-                        alluid = alluId,
-                        applicationIdentifier = identifier,
-                        userId = "user",
-                        hanke = hanke,
-                    )
-                    .withCustomer(ApplicationFactory.createCompanyCustomerWithOrderer())
-            )
+            val hakija = hankeKayttajaFactory.saveUser(hankeId = hanke.id)
+            hakemusFactory
+                .builder(USERNAME, hanke)
+                .withStatus(ApplicationStatus.HANDLING, alluId, identifier)
+                .hakija(yhteyshenkilot = arrayOf(hakija))
```

Comment:
There's an overloaded method that accepts just the hankekayttajat.
```suggestion
                .hakija(hakija)
```